wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/provision/env/WorkspaceAgentJavaOptsEnvVariableProvider.java

```diff
@@ -19,8 +19,8 @@
 public class WorkspaceAgentJavaOptsEnvVariableProvider implements EnvVarProvider {
 
   /** Env variable for jvm settings */
-  public static final String CHE_WORKSPACE_WSAGENT_JAVA_OPTIONS =
-      "CHE_WORKSPACE_WSAGENT_JAVA_OPTIONS";
+  public static final String CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS =
```

Comment:
I did see any reason to name variable in the same way as its value. I'd rename it to `WSAGENT_JAVA_OPTIONS_ENV`